pycbc/psd/__init__.py

```diff
@@ -204,24 +204,28 @@ def insert_psd_option_group(parser, output=True, include_data_options=True):
                                   "to you, then it is probably safe to "
                                   "ignore this option.")
     # Options for PSD variation
+    psd_options.add_argument("--psdvar-segment", type=float,
+                             metavar="SECONDS", help="Length of segment "
+                             "when calculating the PSD variability.")
```

Comment:
might be clearer to say eg 'Length of segment for mean square calculation of PSD variation' to link this explicitly to the mean-square method